manifests/config_entry.pp

```diff
@@ -0,0 +1,36 @@
+# == Define: ssh::config_entry
+#
+# Manage an entry in ~/.ssh/config for a particular user.  Lines model the lines
+# in each Host block.
+define ssh::config_entry (
```

Comment:
@Phil-Friderici we could also use spec tests. This is for our friends in NY.

Reply:
will be my pleasure